docs/source/install.rst

```diff
@@ -1,6 +1,18 @@
 Installation
 ============
 
+Easy
+~~~~
+
+Use ``pip`` or ``conda`` to install::
+
+   $ pip install knit --upgrade
+   $ conda install knit
```

Comment:
Request that we place `or` between these two lines.  Also, should we put conda first?